mne/time_frequency/tests/test_csd.py

```diff
@@ -550,3 +550,23 @@ def test_equalize_channels():
 
     assert csd1.ch_names == ['CH1', 'CH2']
     assert csd2.ch_names == ['CH1', 'CH2']
+
+
+def test_compute_csd():
+    """Test computing cross-spectral density on time-frequency epochs."""
+    rng = np.random.default_rng(11)
+    n_epochs = 6
+    info = mne.io.read_info(raw_fname)
+    info = mne.pick_info(info, mne.pick_types(info, eeg=True))
+    freqs = np.arange(38, 40)
+    times = np.linspace(0, 1, int(round(info['sfreq'])))
+    data = rng.normal(
+        size=(n_epochs, len(info.ch_names), times.size)) * 1e-6
+    epochs = mne.EpochsArray(data, info)
+    csd_test = csd_morlet(epochs, freqs, n_cycles=7)
+    epochs_tfr = tfr_morlet(epochs, freqs, n_cycles=7,
+                            average=False, return_itc=False,
+                            output='complex')
+    csd = compute_csd(epochs_tfr)
+    assert_allclose(csd._data, csd_test._data, atol=1e-16)
```

Comment:
I didn't like this `atol` as it's a bit high. 1e-6 data will have covariance values of 1e-12 (squaring), and for CSD it might be even lower (power within a band). Locally at least it worked with no `atol` param, which should then use the defaults of `atol=0, rtol=1e-7` which seems cleaner to me. Let's see if it comes back green....